packages/components/src/Actions/Action/Action.component.js

```diff
@@ -8,29 +8,15 @@ import ActionDropdown from '../ActionDropdown';
 import ActionIconToggle from '../ActionIconToggle';
 import Inject from '../../Inject';
 
-/**
- * @typedef {(Object|Function)} Component
- */
+const DISPLAY_MODE_FILE = 'file';
+const DISPLAY_MODE_DROPDOWN = 'dropdown';
+const DISPLAY_MODE_SPLIT_DROPDOWN = 'splitDropdown';
+const DISPLAY_MODE_ICON_TOGGLE = 'iconToggle';
```

Comment:
I externalized those const because they were used with `Action. DISPLAY_MODE_FILE ` before it was set in Actions (in the code, not in runtime)